### PR TITLE
Features/movie-details page structure and styling + Rating component

### PR DESCRIPTION
**Pull request by bedicooper**
I still need to add fetching data logic.

I made a separate common/Rating component that we can import into other components:

```
    <Rating
        voteCount={'335'}
        ratingValue={'7,8'}
        isOnBackdrop={true}
        isOnMainTile={false}
    />
```
`voteCount `and `ratingValue` would be passed from data downloaded from the API

for props `isOnBackdrop `and `isOnMainTile `  - we pass these booleans ​​only in movieDetails Page because there the Rating is styled completely differently than in other places in the app.

### CODE REVIEW

**Comment by bedicooper** [comment]
@widmo200 I urgently need your review to merge this PR

File: src/features/movies/MovieDetails/styled.js
Comment (patrycja-gomza) [inline src/features/movies/MovieDetails/styled.js:1]: I would suggest to use references to theme.js (breakpoints) instead of media(max-width).

**Review by patrycja-gomza — APPROVED** [review]
Wow, great job! I am impressed by the precise and detailed work done in this PR! 👏 
 
What I especially like:
- Keeping the code clear and organized!
- Optimal use of Redux with Sagas.
- Organization of components and styles in styled components.
- Effective use of React features.

Thank you for your solid work and professionalism 💯

File: src/features/movies/MovieDetails/styled.js
Comment (bedicooper) [reply src/features/movies/MovieDetails/styled.js:1]: Thanks for the suggestion - I will do it in next PR as I think I might need to add some breakpoints to theme.

**Pull request merged**